src/Concurrency/Info.hpp

```diff
@@ -17,7 +17,10 @@
  *  @brief Abstraction of information on executor environments
  */
 
+#ifdef _OPENMP
 #include <omp.h>
```

Comment:
Use
```
#include "OpenMP.h"
```
since  the file is in the same folder now.

Reply:
Good one, I cleaned some redundant lines in that file as well.